README_BUILD.md

```diff
@@ -1,45 +1,56 @@
+* Git client needs to be version 2.2.0 or above
 
-# Run the build and tests (this will remove all puppet types and build them again)
-bundle exec rake
-~~~~
+## Setup
 
-## Rake tasks
-You can use following rake tasks for your convenience
+To build the dsc module, you need to clone the source from our github repo and bundle install all the gem dependencies.
 
-~~~
-rake dsc:build             # Import and build all
-rake dsc:clean             # Cleanup all
-rake dsc:dmtf:clean        # Cleanup DMTF CIM MOF Schema files
-rake dsc:dmtf:import       # Import DMTF CIM MOF Schema files
-rake dsc:resources:clean   # Cleanup DSC Powershell modules files
-rake dsc:resources:import  # Import DSC Powershell modules files
-rake dsc:types:build       # Build DSC types in (lib/puppet/type)
-rake dsc:types:clean       # Cleanup DSC types in (lib/puppet/type)
+~~~bash
+# clone the dsc repo
+git clone https://github.com/puppetlabs/puppetlabs-dsc
+# change directory to the code
+cd puppetlabs-dsc
+# install the needed ruby gems/libs
+bundle install --path .bundle/gems --without system_tests
 ~~~
 
-## Building
+*NOTE*: This guide assumes that you have a working ruby installation with bundler installed. The latest version of ruby is assumed, as of this writing this guide was tested with 1.9.3 and 2.3.1.
+
+## Considerations/Known Limitations
 
 When building the types and specs, keep the following considerations in mind:
 
-* Resources must be built with a non-Windows computer due to limitations in the gems that the builder uses to generate types and specs.
-* You must use Ruby 1.9.3 to build the types, again due to a limitation in the builder dependencies.
 * The builder requires a `MOF` and `PSD1` file to build a type. Both need to be present for the build to be successful.
 * The `MOF` file encoding should be UTF-8. This ensures the most compatibility with the mof gem.
-* Git client needs to be version 2.2.0 or above
 
-### The DSC tag file
+## Building Puppet Types from Microsoft DSC Resources
+
+To download all the Microsoft DSC Resources from the [PowerShell/DSCResources](https:/github.com/powershell/DSCResources) repo, parse, and build puppet types from them, run the following command:
+
+~~~bash
+# Run the build (this will remove all puppet types and build them again)
+bundle exec rake dsc:build
+~~~
+
+As you continue to work with the repo, you can clean your working copy and re-build by issuing the following commands:
 
-The DSC tag file, called `dsc_resource_release_tags.yml`, determines which versions of the DSC Resources to build.  If a DSC resource is not listed in the file, the latest version that has been released to the PowerShell gallery will be used during the build, and then added to the `dsc_resource_release_tags.yml` file. An example file is shown below:
+~~~bash
+# Run the build (this will remove all puppet types and build them again)
```

Comment:
This feels redundant. You may as well avoid making a distinction between `build` and `clean then build`. Always clean then build.